ocaml/xapi/updateinfo.ml

```diff
@@ -573,3 +575,33 @@ module UpdateInfo = struct
         error "Failed to parse updateinfo.xml: %s" (ExnHelper.string_of_exn e) ;
         raise Api_errors.(Server_error (invalid_updateinfo_xml, []))
 end
+
+module HostUpdates = struct
+  type t = {
```

Comment:
I wonder why aren't we using xapi-idl here and doing it by hand. Is it because it doesn't support nested types well?

Reply:
This is for an external interface which defines the JSON content in the response of the HTTP endpoint "/updates". I don't think it is suitable to put it in xapi-idl which I understand is holding the definitions of Toolstack's internal interfaces.